homeassistant/components/mobile_app/sensor.py

```diff
@@ -49,7 +49,7 @@ async def async_setup_entry(
     for entry in entries:
         if entry.domain != ENTITY_TYPE or entry.disabled_by:
             continue
-        config = {
+        config: dict = {
```

Comment:
Are all the keys not strings?
```suggestion
        config: dict[str, Any] = {
```